flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/runtime/operators/join/temporal/TemporalRowTimeJoinOperator.java

```diff
@@ -77,11 +78,10 @@
 public class TemporalRowTimeJoinOperator extends BaseTwoInputStreamOperatorWithStateRetention {
 
     private static final long serialVersionUID = 6642514795175288193L;
-
-    private static final String NEXT_LEFT_INDEX_STATE_NAME = "next-index";
+    @VisibleForTesting static final String NEXT_LEFT_INDEX_STATE_NAME = "next-index";
```

Comment:
we can add  `getter` function and make it `@VisibleForTesting` to avoid exposing the private member